include/sound/sof/stream.h

```diff
@@ -85,8 +85,9 @@ struct sof_ipc_stream_params {
 
 	uint32_t host_period_bytes;
 	uint16_t no_stream_position; /**< 1 means don't send stream position */
```

Comment:
@yaochunhung Isn't the description in git commit message reversed? I mean if "no_stream_position" is set as 1, FW will not send position IPC. Further, this has been deprecated in ABI3.10 (see commit "ASoC: SOF: Intel: hda: fix reset of host_period_bytes"). Or do you mean something else with "host stream position is updated" ? I'll add more comments to the FW side PR, maybe it's easier to discuss there.

Reply:
@kv2019i  Thanks for comment, I think commit message should be wrong and need to be modified as The host stream position is updated when no_stream_position is set as "0".  I think the commit "ASoC: SOF: Intel: hda: fix reset of  host_period_bytes" which deprecated no_stream_position in ABI3.10 only influences intel's platforms. For mediatek's platform, it is set as default 0 and DSP fw will update host position periodically.